### ISSUE TITLE
Move language files away from ext:lang

### ISSUE THREAD

**Issue by marcusmohr** [beechit/fal_securedownload#169]
Hey there,
the extension causes a problem at the "Access" tab when editing a file in the Filelist. Therefore some items have no labels unless we uninstall the extension.

The label setting references "ext:lang" which could be the cause since this is [deprecated](https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/9.3/Deprecation-84680-MoveLastLanguageFilesAwayFromExtlangAndRemoveExtlangCompletely.html)

```
'fe_groups' => [
            'exclude' => false,
            'label' => 'LLL:EXT:lang/Resources/Private/Language/locallang_general.xlf:LGL.fe_group',
```

![grafik](https://user-images.githubusercontent.com/32599063/142387403-6903d325-851e-46c8-9484-1a3e26b6e090.png)

**Comment by joey-beech-it** [comment]
@marcusmohr Thanks, we will make sure this will get fixed for the v11 update in the future.

If you would like it fixed before that, a pull request is always welcome.

**Comment by LeoniePhiline** [comment]
https://github.com/beechit/fal_securedownload/pull/171

**Comment by FamousWolf** [comment]
This has been fixed in #170. We will probably release a new version at the end of the week.
